### PR TITLE
Set the baseURL for nested objects

### PR DESCRIPTION
**Pull request by mdiep**
This has been biting us in GitHub Desktop.

### CODE REVIEW

**Comment by mdiep** [comment]
CI fails because CI is up the butt. (#282)

File: OctoKit/OCTObject.m [hunk 1]
```diff
@@ -85,6 +85,23 @@ - (void)setBaseURL:(NSURL *)baseURL {
 		NSString *baseURLString = [NSString stringWithFormat:@"%@://%@", baseURL.scheme, baseURL.host];
 		self.server = [OCTServer serverWithBaseURL:[NSURL URLWithString:baseURLString]];
 	}
+	
+	// Also set the base URL for any nested objects.
+	for (NSString *propertyKey in self.class.JSONKeyPathsByPropertyKey) {
```
Comment (joshaber) [inline OctoKit/OCTObject.m]: Objective-C at its best/worst.

**Comment by joshaber** [comment]
Seems legit :+1:

**Comment by joshaber** [comment]
#jolo

**Pull request merged**